layers/cmdline.py

```diff
@@ -0,0 +1,37 @@
+import argparse
+from exporters.to_svg import ToSvg
+from exporters.to_excel import ToExcel
+from core import Layer
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Export an Att&ck layer as a svg image or excel file')
+    parser.add_argument('-m', '--mode', choices=['svg', 'excel'], required=True, help='The form to export the layers in')
```

Comment:
Needs to check the extensions of the specified output to make sure they match the `--mode` selection.

Reply:
I'll put that safety in.